marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collectorLib.sjs

```diff
@@ -39,11 +43,25 @@ class CollectorLib {
       return fn.normalizeSpace(`${sourceQuery}`);
     }
 
+    //Get sourceQuery from mapping artifact if present
+    if(combinedOptions.mapping) {
```

Comment:
Whoa - why are we supporting another location for the sourceQuery? Why would it be on the mapping?